src/components/aktivitetskrav/vurdering/VurderAktivitetskravButtons.tsx

```diff
@@ -29,6 +35,14 @@ export const VurderAktivitetskravButtons = ({
       <Button variant="secondary" onClick={() => onButtonClick("OPPFYLT")}>
         {texts.oppfylt}
       </Button>
+      {aktivitetskrav && toggles.isSendingAvForhandsvarselEnabled && (
```

Comment:
Betyr dette at man ikke kan sende forhåndsvarsel uten eksisterende aktivitetskrav, som man kan med de andre? Hvordan blir det for NAV utland?

Reply:
Jepp, per idag blir det slik!
Men planen er å implementere enda et endepunkt i backenden som støtter å sende forhåndsvarsel uten at det eksisterer et aktivitetskrav, slik man kan for de andre vurderingene 👍🏼